spec/models/activity_spec.rb

```diff
@@ -1,9 +1,12 @@
 require 'rails_helper'
 
 describe Activity do
+
   it { should have_and_belong_to_many(:users) }
   
-  it 'is not being saved without a name'
-  it 'will not be saved if there is a activity with the same name '
+  subject { create(:activity) }
```

Comment:
we do not actually need a created instance here. It should work just fine even without that. As subject by default is always a new instance of described_class which is in this case `Activity`